packages/grpc-js-core/test/test-call-stream.ts

```diff
@@ -103,16 +103,16 @@ describe('CallStream', () => {
     assert2.afterMustCallsSatisfied(done);
   });
 
-  it('should end a call with an error if a stream was closed', (done) => {
+  describe('should end a call with an error if a stream was closed', () => {
     const c = http2.constants;
     const s = Status;
     const errorCodeMapping = {
-      [c.NGHTTP2_NO_ERROR]: s.INTERNAL,
+      [c.NGHTTP2_NO_ERROR]: s.OK,
```

Comment:
This change isn't correct. A `NO_ERROR` HTTP/2 `RST_STREAM` code maps to an `INTERNAL` gRPC status code because closing a stream with a `RST_STREAM` frame instead of a trailers frame with the proper information is considered a gRPC protocol error.

Reply:
Ok -- this change was made in response to [this `case` addition](https://github.com/grpc/grpc-node/blame/master/packages/grpc-js-core/src/call-stream.ts#L274-L276) in a previous PR, which was done to support the JS implementation passing tests. I've gone back to see exactly which test case failed because NO_ERROR was mapped to INTERNAL, but couldn't reproduce it. So I'll change it back as you mentioned and remove that `case`.